HedgeModManager/Languages/pt-PT.xaml

```diff
@@ -13,15 +13,15 @@
     <system:String x:Key="CommonUIDelete">Apagar</system:String>
     <system:String x:Key="CommonUINone"  >Nenhum</system:String>
     <system:String x:Key="CommonUIIgnore">Ignorar</system:String>
-    <system:String x:Key="CommonUIWarning">AVISO</system:String>
```

Comment:
Keep "AVISO" (Full Caps)

Reply:
I didn't know that, we keep the caps then